src/main/c/Jep/convert_p2j.c

```diff
@@ -729,9 +768,9 @@ static int pybufferview_check_jbytearray(Py_buffer view)
 static jbyteArray pybuffer_as_jbytearray(JNIEnv *env, PyObject *pybuffer)
 {
     jbyteArray jarr = NULL;
-    Py_buffer view;
+    JepBuffer view;
 
-    if (PyObject_GetBuffer(pybuffer, &view, PyBUF_FULL_RO) < 0) {
+    if (JepBuffer_FromObject_ReadOnly(pybuffer, &view) < 0) {
```

Comment:
Here is another API we end up emulating: The buffer API.

Turns out the buffer API provides a nice interface to dealing with bytes-like objects in general. We lose that when we use the standard API.